frontend/nginx-conf/diamonds.conf

```diff
@@ -23,4 +31,19 @@ server {
     location /metrics {
         proxy_pass http://backend:5000/metrics;
     }
+
+}
+
+server {
+    listen 443 ssl;
+    server_name diamonds.etimo.se;
+
+    location / {
+        proxy_pass http://diamonds.etimo.se/; #for demo purposes
```

Comment:
Can you use proxy_pass for external url?

Reply:
I have removed this.